src/Utility/MathUtil.ts

```diff
@@ -74,11 +74,7 @@ export default class MathUtil {
     }
 
     static vectorLength(v1) {
-        return MathUtil.length(v1.x, v1.y);
-    }
-
-    static length(x, y) {
-        let len = Math.pow(x, 2) + Math.pow(y, 2);
+        let len = Math.pow(v1.x, 2) + Math.pow(v1.y, 2);
```

Comment:
I think I can revert these changes using one of a couple changes:

1. come up with a new name for the `length` method
2. convert this class to module syntax and leave `length` as is

I think option two is the best because:

- it will leave the interface of the module unchanged
- classes with all static method are considered anti-pattern in JS because they mimic what can be done using module syntax.